workflow-templates/add-new-issues-to-a-project.yml

```
@@ -10,7 +10,7 @@ on:
     types: [opened]
 
 jobs:
-  Add an issue to project:
+  Add_an_issue_to_project:
```

```suggestion
  add-an-issue-to-project:
```
I have no source for this, but I think I'm used to seeing these written in snake-case.

---

you're right Carlos. I updated my suggestion.